viz/MLSMapVisualization.cpp

```diff
@@ -597,8 +597,9 @@ void MLSMapVisualization::visualize(vizkit3d::PatchesGeode& geode, const Surface
     minZ -= 5e-4f;
     maxZ += 5e-4f;
     Eigen::Vector3f normal = p.getNormal();
-    if(normal.z() < 0)
-        normal *= -1.0;
+    //if(normal.z() < 0)
```

Comment:
Can you remove these lines, when they are not needed anymore?

Reply:
the commented lines are deleted